### DISCUSSION TITLE
Add commas for thousand separator

### DISCUSSION THREAD

**Discussion by lulunac27a** [johnkerl/miller#1488] [Ideas]
I want to format the numeric output with commas (thousand separators). For example: 
1337 is 1,337
1234567 is 1,234,567

**Comment by johnkerl** [comment]
I would _love_ it if I could support `"%'d"` within Miller's `fmtnum` -- however in the implementation of `fmtnum` I leverage the Go library's `fmt.Sprintf` which does _not_ accept `"%'d"` :(

**Comment by johnkerl** [comment]
I am working on this in issue #1490
